pkg/resource/models/service_private_endpoints_attachment.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+type ServicePrivateEndpointsAttachment struct {
+	PrivateEndpointIds types.List   `tfsdk:"private_endpoint_ids"`
```

Comment:
```suggestion
	PrivateEndpointIDs types.List   `tfsdk:"private_endpoint_ids"`
```

This is kinda golang way to use abbreviations in var/fields names

Reply:
done